### PR TITLE
Readme Update and Hello World example

### PR DESCRIPTION
**Pull request by Thacryba**

### CODE REVIEW

**Review by ixje — CHANGES_REQUESTED** [review]
I think overall we should have a close look at the `Supported Features` list and make sure it's up to date. Currently `neo3-boa` is capable of more than what the readme suggests, which is doing the project a disservice.

File: README.md [hunk 1]
```diff
@@ -23,77 +48,141 @@ The `neo3-boa` compiler is a tool for compiling Python files to the `.nef` and `
...
+|🔜 | <p>Chained assignment<p/>                 | <p>`x = y = foo()`</p> |
+|✅ | <p>Sequence Slicing<p/>                                        | <p>`x = 'example'[2:4]`, `x = [1, 2, 3][:2]`, `x = 'example'[4:]`, `x = (1, 2, 3)[:]`, `x = 'example'[-4:-2]`, `x = 'example'[:-4]`</p> |
+|🔜 | <p>Sequence Slicing</p> | <p>`x = 'example'[2:4:2]`, `x = 'example'[::2]`</p> |
+|🔜 | <p>Bytes Slicing</p> | <p>`x = b'\x01\x02\x03\x04'[2:]`, `x = b'\x01\x02\x03\x04'[:]`, `x = b'\x01\x02\x03\x04'[::2]`</p> |
+|✅ | <p>Assert</p>                                             | <p>`assert x % 2 == 0`<br/>`assert x % 3 != 2, 'error message'`</p> |
+|🔜 | <p>`continue`</p>                                            |  |
+|🔜 | <p>`break`</p>                                               |  |
+|🔜 | <p>`pass`</p>                                                |  |
+|✅ | <p>Import</p>                                                | <p>`from boa3.builtin import public`<br/>`from typing import *`</p> |
+|🔜 | <p>Import</p>                                                | <p>`import os`<br/>`from path.to.your.package import *`</p> |
```
[the remark this answers is not in the document]
Comment (meevee98) [reply README.md:159]: Only boa3 packages and some symbols in the typing library are supported for now. But I agree, it's confusing in the README which imports are supported and which don't.

**Comment by meevee98** [comment]
> I think overall we should have a close look at the `Supported Features` list and make sure it's up to date. Currently `neo3-boa` is capable of more than what the readme suggests, which is doing the project a disservice.

This list was supposed to inform which Python features are implemented. That's why some features, like Neo interops, manifest options, and debugging support aren't listed there.
@ixje do you think we should include these features or just change the title for `Python Supported Features` or something like this?

**Comment by ixje** [comment]
Yes I agree. Either we list all features, or rename it to something like “supported Python language subset”.

**Pull request merged**